src/scripts/browserDeviceProfile.js

```diff
@@ -294,12 +294,55 @@ import browser from './browser';
                     (browser.tizen && isTizenFhd ? 20000000 : null)));
     }
 
+    function getSpeakerCount() {
+        const AudioContext = window.AudioContext || window.webkitAudioContext || false; /* eslint-disable-line compat/compat */
+
+        if (AudioContext) {
+            const audioCtx = new AudioContext();
+
+            return audioCtx.destination.maxChannelCount;
+        }
+
+        return -1;
+    }
+
+    function getPhysicalAudioChannels(options) {
+        const allowedAudioChannels = parseInt(userSettings.allowedAudioChannels() || '-1');
```

Comment:
`allowedAudioChannels` has `-1` fallback value and use an explicit radix to be sure.
```suggestion
        const allowedAudioChannels = parseInt(userSettings.allowedAudioChannels(), 10);
```

Reply:
done, added radix